### DISCUSSION TITLE
DS4 and DS TouchPad working as Start/Select

### DISCUSSION THREAD

**Discussion by CorrePlayer** [darthcloud/BlueRetro#916] [Ideas]
Could you make the left side of the TouchPad work as Select and the right side work as Start when using Dualshock 4 or Dualsense in Playstation or Playstation 2?

**Comment by ButtonBasher92** [comment]
Would like to see this implemented too. was the first thing i tried to map when I received my BlueRetro but seems the touchPad is only detected as one button.

**Comment by darthcloud** [comment]
It's possible but It's the kind of stuff that I would like someone to submit a PR for.
